sdk/data/azcosmos/cosmos_partition_key_range_response.go

```diff
@@ -0,0 +1,63 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package azcosmos
+
+import (
+	"encoding/json"
+	"net/http"
+
+	azruntime "github.com/Azure/azure-sdk-for-go/sdk/azcore/runtime"
+)
+
+// PartitionKeyRange represents a single partition key range from GET /pkranges endpoint
+// Includes all the needed fields to represent a partition key range
+type PartitionKeyRange struct {
+	ID                      string   `json:"id"`
+	Rid                     string   `json:"_rid"`
```

Comment:
We usually use the term `ResourceID` for this in public properties, rather than the abbreviation RID